.storybook/manager.js

```diff
@@ -1,4 +1,4 @@
-import { addons } from '@storybook/addons';
+import { addons } from '@storybook/manager-api';
 import OrangeTheme from 'ods-storybook-theme/OrangeTheme.js';
```

Comment:
There seems to be an issue with the theme where the newly created `barHoverColor` in Storybook isn't available.